www/js/app.js

```diff
@@ -3336,4 +3337,19 @@ function goToMainArticle () {
     });
 }
 
+function handlePrintButton() {
+    // Attach print function to button
+    const printBtn = document.getElementById('printButton');
+    if (printBtn) {
```

Comment:
The print button is a static part of the app, so you don't have to test for its existence.